shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -199,18 +201,16 @@ export default {
       set(this.value.spec, 'rkeConfig.machineSelectorConfig', [{ config: {} }]);
     }
 
-    // Store the initial PSP template name, so we can set it back if needed
-    const lastDefaultPodSecurityPolicyTemplateName = this.value.spec.defaultPodSecurityPolicyTemplateName;
-    const previousKubernetesVersion = this.value.spec.kubernetesVersion;
-
     const truncateLimit = this.value.defaultHostnameLengthLimit;
+    const previousKubernetesVersion = this.value.spec.kubernetesVersion;
```

Comment:
previousKubernetesVersion and lastDefaultPodSecurityPolicyTemplateName have just moved position